spec/services/git_based_domain_import_service_spec.rb

```diff
@@ -1,66 +1,87 @@
 describe GitBasedDomainImportService do
-  describe "#import" do
-    let(:git_repo) { double("GitRepository", :git_branches => git_branches, :id => 123) }
-    let(:domain) { double("MiqAeDomain") }
+  let(:git_repo) { double("GitRepository", :git_branches => git_branches, :id => 123) }
+  let(:domain) { double("MiqAeDomain") }
+  let(:user) { FactoryGirl.create(:user) }
+  let(:task) { FactoryGirl.create(:miq_task) }
```

Comment:
Same thing here as the above `FactoryGirl`, we don't actually need to create an `miq_task`, it's just unnecessary database hits.